sections/footer.liquid

```diff
@@ -73,14 +73,36 @@
                       {%- endfor -%}
                     </ul>
                   {%- endif -%}
+                {%- when 'brand_information' -%}
+                  <div class="footer-block-brand-information">
+                    {%- if settings.brand_image != blank -%}
+                      {%- assign image_size_2x = settings.brand_image.width | times: 2 | at_most: 5760 -%}
+                      <img
+                        srcset= "{{ settings.brand_image | image_url: width: settings.brand_image.width }}, {{ settings.brand_image | image_url: width: image_size_2x }} 2x"
+                        src="{{ settings.brand_image | image_url: width: 760 }}"
+                        alt="{{ settings.brand_image.alt | escape }}"
+                        class="global-media-settings"
+                        loading="lazy"
+                        width="{{ settings.brand_image.width }}"
+                        height="{{ settings.brand_image.height }}"
+                        style="max-width: min(100%, {{ settings.brand_image_width }}px);"
+                      >
+                    {%- endif -%}
+                    <h2 class="footer-block__heading">{{ settings.brand_headline }}</h2>
+                    <p>{{ settings.brand_description }}</p>
+                    {%- if block.settings.show_social -%}
+                      {%- render 'social-icons' -%}
+                    {%- endif -%}
+                  </div>
                 {%- when 'image' -%}
                   <div class="footer-block__details-content footer-block-image {{ block.settings.alignment }}">
                     {%- if block.settings.image != blank -%}
                       {%- assign image_size_2x = block.settings.image_width | times: 2 | at_most: 5760 -%}
                       <img
                         srcset= "{{ block.settings.image | image_url: width: block.settings.image_width }}, {{ block.settings.image | image_url: width: image_size_2x }} 2x"
-                        src="{{ block.settings.image | image_url: width: 400 }}"
+                        src="{{ block.settings.image | image_url: width: 760 }}"
                         alt="{{ block.settings.image.alt | escape }}"
+                        class="global-media-settings"
```

Comment:
Pseudo elements don't work on `image` elements. So only the border radius and border work. Is that expected or should the shadow be ideally coming up ?
<details>
<summary>Screenshot</summary>

[docs](https://developer.mozilla.org/en-US/docs/Web/CSS/::after)
![](https://screenshot.click/02-19-w91qb-475m7.png)
</details>